src/schema.js

```diff
@@ -165,6 +165,26 @@ export default class DataStructureGenerator {
       .value();
   }
 
+  /** Validates that the given schema matches the given type
+   *
+   * In the case where there is no provided type, the allOf types are matched.
+   */
+  validateSchemaTypes(schema, type) {
+    if (schema.type === type) {
+      return true;
+    }
+
+    if (schema.type === undefined && schema.allOf && schema.allOf.length > 0) {
+      const schemasWithoutMatchingType = schema.allOf.filter((subschema) => {
```

Comment:
I can't win with this line. I wanted to use what I have, but it produces a lint error:

```
/Users/kyle/Projects/apiaryio/fury-adapter-swagger/src/schema.js
  178:77  error  Unexpected block statement surrounding arrow body  arrow-body-style
```

It wants me to make this a one-liner, but then its a ridiculously long one-liner. If changed you'd then get another lint error that the line is too long. So you can't win:

```
/Users/kyle/Projects/apiaryio/fury-adapter-swagger/src/schema.js
  178:1  error  Line 178 exceeds the maximum line length of 100  max-len
```

eslint 🤦‍♂️. Anyone have a suggestion about this dumb `arrow-body-style` rule?

Reply:
Made eslint disable `arrow-body-style` for the file because that rule is dumb, IMO it should be down to code-write on what makes sense for given code block.

Reply:
Can't help it since we are using airbnb as base